reactioncommerce/reaction
# 2181 Revision Control Option Images

Resolves #2181

**Testing**
- Clone this branch and run on your local machine
- Log in as an Admin and add image for a product's option without publishing
- Open an incognito tab and add that option to cart
- Click on the `cart` icon
- Observe that no image is displayed for the option since it hasn't been published by admin
